### PR TITLE
add: Display rendered System and User prompts

### PR DESCRIPTION
**Pull request by mrshu**
* Ensure that when jinja2 is used, the rendered versions of the System and User prompts are also shown in the UI.

### CODE REVIEW

File: prompterator/main.py [hunk 1]
```diff
@@ -470,6 +470,26 @@ def set_up_prompt_attrs_area(st_container):
             height=c.DATA_POINT_TEXT_AREA_HEIGHT,
         )
 
+        system_prompt = env.from_string(st.session_state.system_prompt).render(**st.session_state.row.to_dict())
+        user_prompt = env.from_string(st.session_state.user_prompt).render(**st.session_state.row.to_dict())
+        # Show the rendered System and User prompts
+        col1, col2 = st.columns([1, 1])
```
Comment (mrshu) [inline prompterator/main.py]: @samsucik @sweco @danielajass I am honestly not sure if I like it this way, it does bloat the UI quite a bit. It did seem much better than not seeing the rendered version at all :)

File: prompterator/main.py [hunk 2]
```diff
@@ -470,6 +470,30 @@ def set_up_prompt_attrs_area(st_container):
             height=c.DATA_POINT_TEXT_AREA_HEIGHT,
         )
 
+        system_prompt = env.from_string(st.session_state.system_prompt).render(
```
Comment (mrshu) [inline prompterator/main.py]: @samsucik @sweco this is the very same thing as in:

https://github.com/slidoapp/prompterator/blob/280406f528c613cf23a86f8cc57f2243c60f834c/prompterator/utils.py#L228-L229

I guess if we like it, we could abstract it out somewhere

**Review by sweco — COMMENTED** [review]
This definitely helps, but there was some bug - if was not displaying unless your prompt looked somehow. But I remember having jinja there but it was still not showing. We need to test this a bit more.

**Comment by samsucik** [comment]
@sweco @vikion I made quite some changes to Marek's first version and I'm curious what you think of it -- not necessarily of the code but more of the actual user experience 🙂 Could you give it a try, ideally with a CSV file that has more than just the `text` column and also with prompts that use columns beyond `text`?

**Pull request merged**